GW2EIEvtcParser/EIData/CombatReplay/CombatReplay.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using GW2EIEvtcParser.EncounterLogic;
```

Comment:
?

Reply:
I was checking something out and visual studio added it without me noticing, I'll remove it